src/components/ble/QrService.cpp

```diff
@@ -0,0 +1,75 @@
+#include "QrService.h"
+#include "systemtask/SystemTask.h"
+
+using namespace Pinetime::Controllers;
+
+int QRCallback(uint16_t conn_handle, uint16_t attr_handle, struct ble_gatt_access_ctxt *ctxt, void *arg) {
+  auto qrService = static_cast<QrService *>(arg);
+  return qrService->OnCommand(conn_handle, attr_handle, ctxt);
+}
+
+QrService::QrService(Pinetime::System::SystemTask &system) : m_system(system) {
+  qrsTextCharUuid.value[11] = qrsTextCharId[0];
+  qrsTextCharUuid.value[12] = qrsTextCharId[1];
+
+  characteristicDefinition[0] = {.uuid = (ble_uuid_t *) (&qrsTextCharUuid),
```

Comment:
I think these should be updated to the non-c-style cast style. Like other services today.

Reply:
Thanks :+1: Done now